modules/ROOT/partials/configuration/editimage_api_key.adoc

```diff
@@ -0,0 +1,17 @@
+== `+editimage_api_key+`
```

Comment:
This is meant to be an internal/cloud only option from my understanding, so I don't believe this should be documented. If it's not reverted then this should have an id set like so: `[[editimage_api_key]]`

- [ ] This file addition should probably be reverted

Reply:
those new apis can be reviewed with DOC-1574, i've added the anchors for now

Reply:
It's poor practice to merge in partial work especially when we know it shouldn't be there, as it just leaves it open to being forgotten/lost later. I'm not comfortable approving this while this still exists, as I'm 99.99% sure this should not be here.